experimental/packages/opentelemetry-exporter-metrics-otlp-http/src/OTLPMetricExporterOptions.ts

```diff
@@ -15,12 +15,16 @@
  */
 
 import { OTLPExporterConfigBase } from '@opentelemetry/otlp-exporter-base';
-import { AggregationTemporality } from '@opentelemetry/sdk-metrics';
+import {
+  AggregationTemporality,
+  AggregationSelector,
+} from '@opentelemetry/sdk-metrics';
 
 export interface OTLPMetricExporterOptions extends OTLPExporterConfigBase {
   temporalityPreference?:
     | AggregationTemporalityPreference
     | AggregationTemporality;
+  aggregationSelector?: AggregationSelector;
```

Comment:
I think preference may be the better name as it can be overwritten by a view.

```suggestion
  aggregationPreference?: AggregationSelector;
```

Reply:
Done.